### DISCUSSION TITLE
Programming language typing race?

### DISCUSSION THREAD

**Discussion by ryansamaeian** [monkeytypegame/monkeytype#2321] [Ideas] [answered]
Wild idea here, what if we developers could monkeytype in x programming languages. C, C++, Java, x86_64 accembly, haskell, fortan would be some very nice starting options ! Also we could support other languages like Porth. To further recognize the thrilling opensource community ! Best of luck

Yours Truly,
Ryan.

**Comment by rizwanmustafa** [comment, marked as answer]
You can already code in different languages.
Select 'code' from language groups in settings.
Then choose your programming language.

**Reply by Miodec** [reply]
There are also code quotes available (all though limited) for some languages.
